config/settings/base.py

```diff
@@ -384,7 +384,12 @@
 UMB_BREW_MONITOR_ENABLED = strtobool(os.getenv("CORGI_UMB_BREW_MONITOR_ENABLED", "true"))
 
 # Set to True to turn on loading of community products from product-definitions.
-COMMUNITY_MODE_ENABLED = strtobool(os.getenv("CORGI_COMMUNITY_MODE_ENABLED", "false"))
+if CORGI_DOMAIN.endswith(".fedoraproject.org"):
+    COMMUNITY_MODE_ENABLED = True
+    CORGI_DOMAIN_BASE = ".fedoraproject.org"
+else:
+    COMMUNITY_MODE_ENABLED = False
+    CORGI_DOMAIN_BASE = ".redhat.com"
```

Comment:
Unfortunately we can't set the COMMUNITY_MODE_ENABLED flag in this check because stage still uses an internal URL. I didn't request an external cname and SSL cert for community stage at this time. We could optionally request that and have this change work as written. But for now, I think we should change it to just set the CORGI_DOMAIN_BASE setting.

```suggestion
COMMUNITY_MODE_ENABLED = strtobool(os.getenv("CORGI_COMMUNITY_MODE_ENABLED", "false"))

if CORGI_DOMAIN.endwith(".fedoraproject.org"):
    CORGI_DOMAIN_BASE = ".fedoraproject.org"
else:
    CORGI_DOMAIN_BASE = ".redhat.com"
    
```

Reply:
I was planning to set CORGI_DOMAIN myself to some placeholder URL during the deployment, like "corgi-stage-community-url-placeholder.fedoraproject.org". Without a CORGI_DOMAIN ending in .fedoraproject.org, the logic you suggest here for CORGI_DOMAIN_BASE won't work.

I'll just tweak this to keep COMMUNITY_MODE_ENABLED unchanged, then add ```".fedoraproject.org" if COMMUNITY_MODE_ENABLED else ".redhat.com"``` to our ALLOWED_HOSTS. Sound good?

Reply:
Actually having an internal CORGI_DOMAIN will cause other issues. All the links the API generates are going to use this internal CORGI_DOMAIN.

@jasinner Can you please file that ticket to get an external cname and cert for community-stage? We can keep this code as-is, then merge and redeploy with an updated CORGI_DOMAIN env var once the cname is available.

Reply:
Or if you'd prefer, you could open a new MR with just the first commit from this branch and merge that right away. Then we can merge the second commit only once we have the stage cname and cert set up (and I think this fix is less urgent anyway). I need to log off because it's very late - have a good day!

Reply:
The API is deployed on the internal CORGI_DOMAIN for community-stage instance, so I'm going to merge this updated as I suggested here. We can always migrate to an external host for community-stage if required later.